dash/test/algorithm/ConnectedComponentsTest.cc

```diff
@@ -0,0 +1,748 @@
+#include "ConnectedComponentsTest.h"
+#include <dash/Graph.h>
+#include <dash/algorithm/graph/ConnectedComponents.h>
+
+struct vprop {
+  dash::default_index_t comp;
+  dash::global_unit_t   unit;
+};
+
+typedef dash::Graph<dash::UndirectedGraph, vprop>       graph_t;
+
+TEST_F(ConnectedComponentsTest, AlgorithmRun)
+{
+  std::list<std::pair<int, int>> edge_list = {
```

Comment:
Just out of curiosity: how  was this generated (please tell me not by hand ;)) and can we generate it on the fly?

Reply:
It's from a graph sample data set, @stiefn should know more.